bindgen/bindgen.bzl

```diff
@@ -0,0 +1,167 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_rust//rust:rust.bzl", "rust_library")
+
+def rust_bindgen_library(name, header, cc_lib, **kwargs):
```

Comment:
Since this library forward to a rust_library and this file control the behavior of rust_bindgen, why not explicitely list arguments for rust_bindgen and use the kwargs to forward to rust_library?

Reply:
+1 on this. If we're exposing this as a "rust_library", it probably should have the same attrs, plus the bindgen ones.